pants

```diff
@@ -1,91 +0,0 @@
-#!/usr/bin/env bash
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
```

Comment:
Probably need to restore this.

Reply:
whoops! I am having mad trouble running tests locally and re-bootstrapped my pants checkout. This certainly doesn't belong here!